en/yohane.md

```diff
@@ -2,33 +2,33 @@
 title: "Yohane Commands"
 old_id: 4
 ---
-Berikut ini adalah command-command yang ada di bot Yohane.
+These are the commands for the Yohane bot.
 
 ## Yohane In-Game
 
 ### General commands
-- `!roll [angka]` - Memberikan random number
-- `!help` - Memberikan daftar bantuan
-- `!subscribe pp [options]` - Untuk menampilkan status pp seperti limit dan notifikasi jika sudah selesai [submit score](https://cdn.discordapp.com/attachments/265909019976138754/801454622346444860/unknown.png). (options : `set`,`get`,`unset`,`toggle`)
-- `!boardshow [options]` - [Experimental] Untuk menyembunyikan diri dari leaderboard (options : `others`,`normal`)
-- `!report` - Jika kalian menemukan cheater, silahkan gunakan command ini.
-- `!faq list` - Daftar hal yang sering dibutuhkan dalam chat.
+- `!roll [number]` - Returns random number
```

Comment:
```suggestion
- `!roll [number]` - Roll a random set of number
```

Reply:
"set of number" is grammatically incorrect, and as it only returns one number, referring to it as "set" would be wrong

Reply:
I would also suggest to change the argument from "number" with eg "max" (I think it's the maximum amount? It's not really clear based on the given description)

Reply:
Kind of confused on how to word it. I will leave it up to you 🤔

Reply:
I already did updates on the last commit, feel free to check

Reply:
Checked, it's good 😎